Description: Fix FTBFS on GNU/kFreeBSD caused by missing strnstr() on glibc systems.
Author: Fathi Boudra <fabo@debian.org>
Bug: https://bugs.webkit.org/show_bug.cgi?id=33903

---
 src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h |    2 +-
 src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h
+++ b/src/3rdparty/webkit/Source/JavaScriptCore/wtf/Platform.h
@@ -427,7 +427,7 @@
 #endif
 
 /* OS(LINUX) - Linux */
-#ifdef __linux__
+#if defined(__linux__) || defined(__GLIBC__)
 #define WTF_OS_LINUX 1
 #endif
 
--- a/src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
+++ b/src/3rdparty/javascriptcore/JavaScriptCore/wtf/Platform.h
@@ -423,7 +423,7 @@
 #endif
 
 /* OS(LINUX) - Linux */
-#ifdef __linux__
+#if defined(__linux__) || defined(__GLIBC__)
 #define WTF_OS_LINUX 1
 #endif
 
